src/scene/container/RenderContainer.ts

```diff
@@ -86,7 +86,7 @@ export class RenderContainer extends Container implements View, Instruction
     public addBounds: (bounds: Bounds) => void;
 
     public canBundle = false;
-    public renderPipeId = 'customRender';
+    public readonly renderPipeId: string = 'customRender';
```

Comment:
educate an idiot here... but would `public readonly renderPipeId = 'customRender';` not infer a string? thanks!

Reply:
It becomes 

```ts
public readonly renderPipeId: 'customRender' = 'customRender';
```

The `readonly` affects the inferred type: without it we get `string` as expected.

Reply:
ahh! thanks